thoth/common/openshift.py

```diff
@@ -2325,6 +2325,68 @@ def get_kebechet_template(self) -> Dict[str, Any]:
 
         return self._get_template("template=kebechet")
 
+    def schedule_si_bandit(
+        self,
+        python_package_name: str,
+        python_package_version: str,
+        python_package_index: str = "https://pypi.org/simple",
```

Comment:
Do we not want a default here?

Reply:
I would not provide it, it might be misleading. PyPI is just another package index for us, no reason why it should be special in this case. And it might be better to be explicit here.